typescript/tests/createVaas.ts

```diff
@@ -11,17 +11,21 @@ function getFromEnvironment(key: string) {
   );
 }
 
-export default function createVaas(): Promise<Vaas> {
+export default async function createVaas(): Promise<Vaas> {
   dotenv.config();
   const CLIENT_ID = getFromEnvironment("CLIENT_ID");
   const CLIENT_SECRET = getFromEnvironment("CLIENT_SECRET");
   const VAAS_URL = getFromEnvironment("VAAS_URL");
   const TOKEN_URL = getFromEnvironment("TOKEN_URL");
 
-  return CreateVaasWithClientCredentialsGrant(
+  const authenticator = new ClientCredentialsGrantAuthenticator(
     CLIENT_ID,
     CLIENT_SECRET,
     TOKEN_URL,
-    VAAS_URL
   );
+
+  let vaas = new Vaas();
+  let token = await authenticator.getToken()
```

Comment:
const

Reply:
you can just make suggestions
```suggestion
  const token = await authenticator.getToken()
```